jsonld-contexts/c2jn-compound.jsonld

```diff
@@ -1,5 +1,6 @@
 {
   "@context": [
+    "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/weather/jsonld-contexts/weather-compound.jsonld",
```

Comment:
your compound uses a 1.3 core context. this compound has a 1.7 core context. don't know what will happen (and not sure I want to know).

Reply:
Can I just replace the 1.3 by the 1.7 in the weather compound or is it more complicated ?

Reply:
I would feel better if you just included the environment and weather contexts (and not the core context)